qemu/tests/cfg/nvdimm.cfg

```diff
@@ -8,7 +8,7 @@
     only x86_64, ppc64le
     ppc64le:
         required_qemu = [5, )
-        only nvdimm_basic
+        only nvdimm_basic nvdimm_nvml
         # PowerPC guests need to create the persistent memory device manually so far
         nvdimm_ns_create_cmd = "ndctl create-namespace"
         dimm_extra_params = "label-size=256m"
```

Comment:
Should we change it as 256M ?

Reply:
I'm not sure. Would you confirm about it? I can update if you need.

Reply:
Yes, it works with 256M.

Reply:
Updated, pls help review, thanks.